File: Python Programming/problems on numbericals/problem15.py
'''
15.Write a program which accept number from user and print its numbers line. 

	Author : Milind Nalawade
	
	Date : 25/06/2021
	
	Language : Java 14.0.2
	
'''

#=====
#Logic
#=====
def Print(no):
	i=0
	i=-no
	while i!=no+1 :
		print(i,end=" ")
		i=i+1
	
#=============
#Entry point function
#=============

def main():
	
	no=int(input("Enter your number : "))
	Print(no)

#======
#starter
#======
if __name__=="__main__":
	main()	

File: C Programming/problem on numbericals/problem15.c
/*
15.Write a program which accept number from user and print its numbers line. 

	Author : Milind Nalawade
	
	Date : 25/06/2021
	
	Language : C programming
	
*/

#include<stdio.h>

//Logic
void Print(int no)
{
	int i=-no;

	while(i!=no+1)
	{
		printf("%d ",i);
		i++;
	}
}

//Entry point function
int main()
{
	int i=0;
	printf("Enter your number : ");
	scanf("%d",&i);
	Print(i);
}